src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,15 @@
+import { useContext } from 'react';
+import { TodosContext } from '../../TodosContext';
+import { TodoItem } from '../TodoItem';
+
+export const TodoList: React.FC<{}> = () => {
```

Comment:
the same you can delete it
```suggestion
export const TodoList: React.FC = () => {
```